js/serial_backend.js

```diff
@@ -246,8 +246,13 @@ function onOpen(openInfo) {
 
         FC.resetState();
 
-        // request configuration data
+        // request configuration data. Start with MSPv1 and
+        // upgrade to MSPv2 if possible.
+        MSP.protocolVersion = MSP.constants.PROTOCOL_V1;
         MSP.send_message(MSPCodes.MSP_API_VERSION, false, false, function () {
+            if (CONFIG.apiVersion && semver.gte(CONFIG.apiVersion, "1.27.0")) {
```

Comment:
We didn't bump API version up when merging MSPv2. I think we should increase version major to "2.0.0" and do the same change in firmware.

Reply:
That make sense. Increase it in INAV and I'll update the PR.

BTW, what's the long term plan for the MSPv1 protocol? Will it be removed or kept around for the foreseeable future?

Reply:
MSPv1 in the firmware will be kept as long as there are tools that use it (like mwosd, EzGUI etc). As for Configurator - we'll probably remove it after a while (probably a long while).

Reply:
It seems we'll be stuck with supporting MSPv1 for a long time then. I was asking because if MSPv1 was dropped from the FC side, the MSPv2 detection logic would eventually need to be changed (e.g. send MSPv2 1st, fallback to MSPv1 on timeout), but it seems that's not gonna be the case for a long time.

Reply:
MSPv1 doesn't put much maintenance burden on FW side. We don't deprecate old messages, MSPv2 is merely more frexible transport layer, so there's no point in deprecating MSPv1 on firmware side. Configurator is another story - once major part of users will be MSPv2 aware we can drop v1 support in Configurator.